frontend/src/pages/ActivityPage.js

```diff
@@ -14,17 +16,7 @@ import {
   Typography,
 } from '@material-ui/core';
 
-const fakeActivityData = {
-  title: 'Muay Thai Session',
-  location: 'My Garage',
-  desc: 'A fun light paced one hour intro to muay thai.',
-  author: {
-    username: 'Jay',
-    fullname: 'Jay Gohner',
-    userId: 420,
-    avatarURL: 'https://bullmuaythaikrabi.com/wp-content/uploads/2017/09/Bull-Muay-Thai-Krabi-fighters-news_6.jpg'
-  },
-};
+const api_url = 'http://localhost:3001/api';
```

Comment:
The api url seems like it shouldn't be hard-coded. Though I'm not sure what the alternative would be. Maybe in entry in a config file?

Reply:
@th3n1ghtmanc0m3th I did some more research into environment variables with React and it seems if you want them to work in the app, they have to be titled like this: REACT_APP_<variable name>. All environment variables used in React have to start with the REACT_APP_ part.